pkg/master/master.go

```diff
@@ -231,21 +231,15 @@ func (m *Master) InstallAPIs(c *Config) {
 
 	// Install extensions unless disabled.
 	if c.APIResourceConfigSource.AnyResourcesForVersionEnabled(extensionsapiv1beta1.SchemeGroupVersion) {
-		m.thirdPartyStorage = c.StorageDestinations.APIGroups[extensions.GroupName].Default
+		var err error
```

Comment:
I dont think that you will get a compilation error for that. You can use `:=` if at least one of the variables on the left side are new.
Can you try?